tests/sqlparser_common.rs

```diff
@@ -903,6 +903,20 @@ fn parse_in_subquery() {
     );
 }
 
+#[test]
+fn parse_in_unnest() {
```

Comment:
Can you please also add a test for `NOT IN UNNEST` as well?

Reply:
Thanks! I will fix it!